Nodejs/Setup/BuildRelease.ps1

```diff
@@ -243,7 +243,8 @@ $managed_files = (
     "Microsoft.NodejsTools.Npm.dll",
     "Microsoft.NodejsTools.TestAdapter.dll",
     "Microsoft.NodejsTools.PressAnyKey.exe",
-    "Microsoft.NodejsTools.VsLogger.dll"
+    "Microsoft.NodejsTools.VsLogger.dll",
```

Comment:
Let's go ahead and remove Microsoft.Nodejstools.VsLogger references while we're at it.